packages/plugin-ext/src/hosted/browser/hosted-plugin-controller.ts

```diff
@@ -128,10 +147,17 @@ export class HostedPluginController implements FrontendApplicationContribution {
      * Display status bar element for running plugin.
      */
     protected async onHostedPluginRunning(): Promise<void> {
-        this.pluginState = HostedPluginState.Running;
+        this.pluginState = HostedInstanceState.Running;
```

Comment:
maybe the states should be uppercase : Running --> RUNNING

Reply:
Agree